integration/e2e/e2e_test.go

```diff
@@ -211,10 +212,61 @@ var _ = Describe("EndToEnd", func() {
 			nwo.CommitChaincode(network, "testchannel", orderer, chaincode, testPeers[0], testPeers...)
 			nwo.InitChaincode(network, "testchannel", orderer, chaincode, testPeers...)
 
+			By("listing the containers after committing the chaincode definition")
```

Comment:
This is tricky. The test is placed in e2e so we can reduce the number of networks and, hopefully, improve the speed of the tests. I like that.

On the other hand, this is pretty unrelated to the kafka network test - especially since we're de-prioritizing kafka going forward.

Is there a better place to put this within lifecycle that would not extend the execution times greatly? The fact that we are deploying chaincode with docker means they will always take a long time...

Reply:
This one seemed to slip through the cracks. Do we have a place in lifecycle that makes sense?

If not, let's move; otherwise we should probably move.

Reply:
Created a task to move this: https://jira.hyperledger.org/browse/FAB-17425